charts/fluent-bit/values.yaml

```diff
@@ -462,10 +459,12 @@ daemonSetVolumeMounts:
     mountPath: /etc/machine-id
     readOnly: true
 
-args: []
-
 command: []
 
+args:
```

Comment:
I guess the intent is to be explicit here, neither of these is required I believe so may be worth adding a comment.

Reply:
They are required as the config is mounted at a different path.